### PR TITLE
Don't bump the version number for the pinning prs

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: conda_forge_tick/migrators/migration_yaml.py [hunk 1]
```diff
@@ -189,43 +189,9 @@ def migrate(
...
-                        new_lines.append(line)
-
-            with open("meta.yaml", "w") as fp:
-                for line in new_lines:
-                    fp.write(line)
-
-            eval_xonsh("git add .")
-
         if not found_ver:
             return False
```
Comment (beckermr) [inline conda_forge_tick/migrators/migration_yaml.py]: These lines need to go too

**Comment by CJ-Wright** [comment]
@beckermr ready for another round

**Comment by beckermr** [comment]
There are build number bump tests there too. Probs don't need those.

**Comment by CJ-Wright** [comment]
@beckermr fixed!

**Pull request merged**